src/telemetry/pings.yaml

```diff
@@ -73,3 +73,23 @@ daemonsession:
       (Android only) Sent every three hours during an active VPN session.
     daemon_end: >
       Sent after as session ending metric is recorded.
+extensionsession:
```

Comment:
Nit: There were two line breaks, I thought there should be just one (but wasn't clear, sorry). It was updated to be zero.